### ISSUE TITLE
Make search case-insensitive

### ISSUE THREAD

**Issue by AlexDaniel** [carnager/rofi-pass#87]
All that's needed is this:
```diff
-    selected_password="$(list_passwords 2>/dev/null | rofi "${args[@]}")"
+    selected_password="$(list_passwords 2>/dev/null | rofi -i "${args[@]}")"
```

**Comment by carnager** [comment]
not needed, that's what the _rofi function in config file is for.

**Comment by AlexDaniel** [comment]
Sorry, what? What do you mean by not needed? It does not use that function on that particular line. The search is case sensitive without this patch. Are you arguing that it should be case sensitive?

**Comment by carnager** [comment]
ah! The real issue is that i forgot to change "rofi" to "_rofi" in some cases. fixed in 6d469c55069e8a175b497fca99918c3982ede19e

**Comment by AlexDaniel** [comment]
Ah, I see! Thanks! Maybe you can also take a look at https://github.com/carnager/rofi-pass/issues/88.
